src/App.tsx

```diff
@@ -30,6 +32,8 @@ const App = () => {
                     <Route path="/tutor-profile" element={<TutorProfilePage />} />
                     <Route path="/tutorsearch" element={<TutorSearchPage />} />
                     <Route path="/tutordashboard" element={<TutorDashboard />} />
+                    <Route path="/404" element={<NotFoundPage />} />
```

Comment:
Instead of specifying the 404 path, use an asterisk (*) to match anything. The NotFoundPage component will render for all the URLs not specified in routes.